apiserver/resources.go

```diff
@@ -33,7 +33,7 @@ type ResourcesBackend interface {
 	GetPendingResource(applicationID, name, pendingID string) (resource.Resource, error)
 
 	// SetResource adds the resource to blob storage and updates the metadata.
-	SetResource(applicationID, userID string, res charmresource.Resource, r io.Reader) (resource.Resource, error)
+	SetResource(applicationID, userID string, res charmresource.Resource, r io.Reader, incrementCharmModifiedVersion bool) (resource.Resource, error)
```

Comment:
Small nit, often for bools it's clearer to define a type alias and use constants for `IncrementCharmVersion` and `DoNotIncrementCharmVersion`. Just a suggeston to maker the api semantics clearer